### PR TITLE
etcd:Update version to 3.5.13

### PR DESCRIPTION
**Pull request by rohit-satya**
Close https://github.com/karmada-io/karmada/issues/4862

**What type of PR is this?**

**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:
Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```
etcd: update version to 3.5.13
```

### CODE REVIEW

**Comment by liangyuanpeng** [comment]
@rohit-satya Thanks for working it and please follow the https://github.com/karmada-io/karmada/pull/4864/checks?check_run_id=24187866088 to sign DCO.

File: charts/karmada/README.md [hunk 1]
```diff
@@ -207,11 +207,11 @@ helm install karmada-scheduler-estimator -n karmada-system ./charts/karmada
...
-| `cfssl.image.pullSecrets`                 | Image pull secret of the cfssl                                                                                                                                                                                                                | `[]`                                                                                                                                                                                                           |
+| `cfssl.image.pullSecrets`                 | Image pull secret of the cfssl                                                                                                                                                                                                                | `[]`                                                                                                                                                                                                                 |
```
Comment (liangyuanpeng) [inline charts/karmada/README.md:214]: This seems like an unexpected change.

File: charts/karmada/README.md [hunk 2]
```diff
@@ -207,11 +207,11 @@ helm install karmada-scheduler-estimator -n karmada-system ./charts/karmada
...
-| `kubectl.image.pullSecrets`               | Image pull secret of the kubectl                                                                                                                                                                                                              | `[]`                                                                                                                                                                                                           |
+| `kubectl.image.pullSecrets`               | Image pull secret of the kubectl                                                                                                                                                                                                              | `[]`                                                                                                                                                                                                                 |
```
Comment (liangyuanpeng) [inline charts/karmada/README.md:210]: This seems like an unexpected change.

File: charts/karmada/README.md (hunk 1 above)
Comment (rohit-satya) [reply charts/karmada/README.md]: handled

File: charts/karmada/README.md (hunk 2 above)
Comment (rohit-satya) [reply charts/karmada/README.md]: handled

**Comment by RainbowMango** [comment]
/approve

Leave LGTM to @liangyuanpeng for final decision.

Thank you @rohit-satya for doing this.

**Comment by liangyuanpeng** [comment]
/lgtm
Thanks

**Pull request merged**